File: Program_Files/Order.cs
/* Online C# Compiler and Editor */
using System;

public class Order {
    // Function that returns 5 for order of operations test
    public static float orderNumberTest() {
        return 4;
    }

    public static void Main(string[] args) {
        // Perform the math, also calling the function as necessary
        float orderTest = 2 * 3 / orderNumberTest() + 5;
        // Print final results
        Console.WriteLine("2 * 3 / orderNumberTest() + 5 = " + orderTest);
    }
}

File: Program_Files/Order.java
/* Online Java Compiler and Editor */
public class Order {
    // Function that returns 5 for order of operations test
    public static double orderNumberTest() {
        return 4.0;
    }

    public static void main(String []args) {
        // Perform the math, also calling the function as necessary
        double orderTest = 2 * 3 / orderNumberTest() + 5;
        // Print final results
        System.out.printf("2 * 3 / orderNumberTest() + 5 = %f\n", orderTest);
    }
}